mirdata/datasets/filosax.py

```diff
@@ -0,0 +1,512 @@
+        self.chord_changes = (
+            input_dict["chord_changes"] if "chord_changes" in input_dict else [0]
+        )
+        self.num_chord_changes = (
+            input_dict["num_chord_changes"] if "num_chord_changes" in input_dict else 0
+        )
+        self.main_chord_num = (
+            input_dict["main_chord_num"] if "main_chord_num" in input_dict else 0
+        )
+        self.scale_changes = (
+            input_dict["scale_changes"] if "scale_changes" in input_dict else [0]
+        )
+        self.loudness_max_val = (
+            input_dict["loudness_max_val"] if "loudness_max_val" in input_dict else 0.0
+        )
+        self.loudness_max_time = (
+            input_dict["loudness_max_time"]
+            if "loudness_max_time" in input_dict
+            else 0.0
+        )
+        self.loudness_curve = (
+            input_dict["loudness_curve"] if "loudness_curve" in input_dict else [0.0]
+        )
+        self.pitch_average_val = (
+            input_dict["pitch_average_val"]
+            if "pitch_average_val" in input_dict
+            else 0.0
+        )
+        self.pitch_average_time = (
+            input_dict["pitch_average_time"]
+            if "pitch_average_time" in input_dict
+            else 0.0
+        )
+        self.pitch_curve = (
+            input_dict["pitch_curve"] if "pitch_curve" in input_dict else [0.0]
+        )
+        self.pitch_vib_freq = (
+            input_dict["pitch_vib_freq"] if "pitch_vib_freq" in input_dict else 0.0
+        )
+        self.pitch_vib_ext = (
+            input_dict["pitch_vib_ext"] if "pitch_vib_ext" in input_dict else 0.0
+        )
+        self.spec_cent = input_dict["spec_cent"] if "spec_cent" in input_dict else 0.0
+        self.spec_flux = input_dict["spec_flux"] if "spec_flux" in input_dict else 0.0
+        self.spec_cent_curve = (
+            input_dict["spec_cent_curve"] if "spec_cent_curve" in input_dict else [0.0]
+        )
+        self.spec_flux_curve = (
+            input_dict["spec_flux_curve"] if "spec_flux_curve" in input_dict else [0.0]
+        )
+        self.seq_len = input_dict["seq_len"] if "seq_len" in input_dict else -1
+        self.seq_num = input_dict["seq_num"] if "seq_len" in input_dict else -1
+
+
+class Track(core.Track):
+    """Filosax track class
+    # -- YOU CAN AUTOMATICALLY GENERATE THIS DOCSTRING BY CALLING THE SCRIPT:
+    # -- `scripts/print_track_docstring.py my_dataset`
+    # -- note that you'll first need to have a test track (see "Adding tests to your dataset" below)
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        audio_path (str): path to audio file
+        annotation_path (str): path to annotation file
+        midi_path (str): path to MIDI file
+        musicXML_path (str): path to musicXML file
+        pdf_path (str): path to PDF file
+
+    Cached Properties:
+        notes ([Note]): an ordered list of Note objects
```

Comment:
Could you please write that as `(list, Note)`? Yours is not incorrect :) Just to be consistent with the rest of the docs.